src/match/tasks.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+import logging
+import os
+import sys
+
+from celery import Celery
+from celery.signals import after_setup_logger
+from django.utils import timezone
+from django_celery_beat.models import IntervalSchedule
+from django_celery_beat.models import PeriodicTask
+from match import match_status
+from match.controllers.create_match_controller import CreateMatchController
+from match.models import Match
+from person.models import Person
+
+# Get Pear algorithm
+current_dir = os.path.dirname(os.path.abspath(__file__))
+submodule_path = f"{current_dir}/../../pear-algorithm/src"
+sys.path.insert(0, submodule_path)
+from main import main
```

Comment:
for styling, is there a reason this is down below the other code

Reply:
I had to disable the pre-commit-config to force this to happen, this needs to be the very last import that is made because it's in a different directory and to actually change the directory the code before is necessary. If you know of a better way of doing this, I am all for it.

Reply:
ohhh i get it now, if it works it works - I couldn't find a better solution online ☹️